src/main/java/carpet/utils/CarpetProfiler.java

```diff
@@ -31,16 +33,38 @@
     private static int tick_health_elapsed = 0;
     private static TYPE test_type = TYPE.NONE; //1 for ticks, 2 for entities
     private static long current_tick_start = 0;
-    private static final String[] GENERAL_SECTIONS = {"Network", "Autosave", "Async Tasks", "Datapacks", "Carpet"};
-    private static final String[] SCARPET_SECTIONS = {
-            "Scarpet run", "Scarpet events", "Scarpet schedule",
-            "Scarpet command", "Scarpet load", "Scarpet app data", "Scarpet client"
-    };
-    private static final String[] SECTIONS = {
-            "Spawning and Random Ticks", "Ticket Manager","Unloading",
-            "Blocks", "Entities", "Block Entities",
-            "Entities (Client)", "Block Entities (Client)",
-            "Village", "Environment"};
+    private static final ImmutableMap<String, String> GENERAL_SECTIONS = ImmutableMap.of(
```

Comment:
No need to use guava here:

```suggestion
    private static final Map<String, String> GENERAL_SECTIONS = Map.of(
```

Same for the other ones. You can use Map.ofEntries(Entry...) if there's more than the `of()` static methods allow (with static import of `Map.entry()` for convenience).